### ISSUE TITLE
Logging Changes

### ISSUE THREAD

**Issue by velkymx** [ProcessMaker/bpm#1183]
When a user makes a change to a task data or request data I want to log that the data is changed. We are going to use this as the foundation for the comment system. 

<img width="746" alt="screenshot 2019-01-14 10 14 44" src="https://user-images.githubusercontent.com/535300/51131664-3e332680-17e5-11e9-9076-26b0a401791b.png">

- [ ] Add table: comments (id, user_id, poly, subject, body, hidden, created_at, updated_at)
- [ ] Add model: comment
- [ ] Comments should be polymorphic (to be used with tasks, requests, messages, etc)
- [ ] Automatically add comment when data is changed
- [ ] Automatically add comment when task is submitted
- [ ] Unit test to confirm log comments writted

**Comment by velkymx** [comment]
Format ideas:

(avatar) TIMESTAMP USERNAME LOGMESSAGE
(AB) 01/16/2019 7:12AM Admin canceled the request
(AB) 01/16/2019 7:12AM Admin changed the data for Approval task

![screenshot 2019-01-16 06 56 53](https://user-images.githubusercontent.com/535300/51258201-44004780-195e-11e9-9b9c-81ccad158af6.png)

**Comment by velkymx** [comment]
Use the avatar component and link to the user's profile.

**Comment by danloa** [comment]
@velkymx  In the comment system, the table has a body and a subject for every comment. In the line:
(avatar) TIMESTAMP USERNAME LOGMESSAGE
LOGMESSAGE = subject
and with a mouseover the body of the message should be displayed?, Do you agree?

**Comment by velkymx** [comment]
No. No mouse over. ALSO show the logs for tasks in the Request detail under the tabs. So if a task is change (data, status, etc) the log on the Request detail will update

Layout: (AA) TIMESTAMP USERNAME BODY

**Comment by velkymx** [comment]
![screenshot 2019-01-18 14 09 05](https://user-images.githubusercontent.com/535300/51415871-339dc780-1b2c-11e9-8851-0c8d696580bb.png)
